docs/installation-guide.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Installation Guide"
+weight: 10
+---
+
+There are various ways you can use the Generator library to generate whaterver you want in your event driven architecture apps. Find your prefered method below:
+- [Using the AsyncAPI Generator CLI](#1-using-the-asyncapi-generator-cli)
```

Comment:
```suggestion
- [Generator CLI](#generator-cli)
```

Not sure if this URL works now, but I wanted to make sure we shorten this to be succinct